chains/edgenchain/metadata.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: testkey
+    apiUrl: https://edgenscan.io/api
+    family: blockscout
+    name: Edgenscan
+    url: https://edgenscan.io
+chainId: 4207
+deployer:
+  name: LayerEdge
+  url: https://www.layeredge.io/
```

omit trailing slashes
```suggestion
  url: https://www.layeredge.io
```